frontend/tests/controllers/ContestCreateTest.php

```diff
@@ -151,6 +151,23 @@ public function testCreateContestWithInvalidAlias() {
         }
     }
 
+    /**
+     * Check if the plagiarism value is stored correctly in the database
+     * when a contest is created
+     */
+    public function testPlagiarismThresholdValue() {
+        // Create a contest
+        $contestData = \OmegaUp\Test\Factories\Contest::createContest(
+            new \OmegaUp\Test\Factories\ContestParams([
+                'checkPlagiarism' => true,
+            ])
+        );
+        $checkPlagiarismDuplicate = \OmegaUp\DAO\Contests::getByAlias(
+            $contestData['request']['alias']
+        );
+        $this->assertTrue($checkPlagiarismDuplicate->check_plagiarism);
```

Comment:
```suggestion
        $contest = \OmegaUp\DAO\Contests::getByAlias(
            $contestData['request']['alias']
        );
        $this->assertTrue($contest->check_plagiarism);
```

this is not a duplicate of a plagiarism check, but a contest.